schema.sql

```diff
@@ -0,0 +1,14 @@
+DROP DATABASE IF EXISTS photo_gallery;
+CREATE DATABASE photo_gallery;
+
+CREATE TABLE listings (
+    listing_id SERIAL PRIMARY KEY
+);
```

Comment:
Maybe I just need an explanation, but I don't really understand the functionality of the listings table if all it contains is all possible listings. Wouldn't this also be contained within your photos table?

Reply:
The photos table references the listings table and is useful for future tables that will need to reference it, for example, user likes.